modules/layers/src/path-layer/path-layer.js

```diff
@@ -307,6 +307,28 @@ export default class PathLayer extends Layer {
     const {pathTesselator} = this.state;
     attribute.value = pathTesselator.get('pickingColors', attribute.value, this.encodePickingColor);
   }
+
+  clearPickingColor(color) {
+    const pickedPathIndex = this.decodePickingColor(color);
+    const {bufferLayout} = this.state.pathTesselator;
+    const numVertices = bufferLayout[pickedPathIndex];
+
+    let startInstanceIndex = 0;
+    for (let pathIndex = 0; pathIndex < pickedPathIndex; pathIndex++) {
+      startInstanceIndex += bufferLayout[pathIndex];
+    }
+
+    const {instancePickingColors} = this.getAttributeManager().attributes;
+
+    const {value} = instancePickingColors;
+    const endInstanceIndex = startInstanceIndex + numVertices;
+    for (let i = startInstanceIndex; i < endInstanceIndex; i++) {
```

Comment:
Idea: Since setting to all zeros, maybe use [TypedArray.fill](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray/fill)?